requirements.txt

```diff
@@ -10,8 +10,11 @@ restless==2.0.1
 # PyYAML is for contact loading
 PyYAML==3.11
 
-# deployed/hosted with gunicorn
-gunicorn
-dj-static
+# deployed/hosted with waitress
+waitress==0.8.9
+dj-static==0.0.6
+
+# use database strings instead of dicts
+dj-database-url==0.3.0
```

Comment:
I'm assuming this is so that we can store the database connection information in a single environment variable?